src/components/forecastDetailGroup/forecastDetailGroup.scss

```diff
@@ -9,7 +9,9 @@
   .precipProb,
```

Comment:
if your elements have some common styles, maybe you should create separate selector for these styles?